UPGRADE-6.3.md

```diff
@@ -636,6 +636,7 @@ Three newly defined log channels are deprecated and only necessary for backwards
 - \TPkgCmsCoreLog
 - \TPkgCmsCoreLogCleanupCronJob
 - \TPkgCmsCoreLogMonologHandler_Database
+- \TPkgCmsLicenseManager_MTLogin
```

Comment:
We normally can't deprecate things retroactively. As we're a bit off-the record anyway by removing the files without deprecating beforehand, it's OK as an exception.